blocks/cards/cards.css

```diff
@@ -2,22 +2,112 @@
   list-style: none;
   margin: 0;
   padding: 0;
-  display: grid;
-  grid-template-columns: repeat(auto-fill, minmax(278px, 1fr));
-  grid-gap: 16px;
+  display: flex;
+  overflow-x: scroll;
+}
+
+@media (width > 600px) {
+  .cards > ul {
+    overflow-x: unset;
+  }
 }
 
 .cards > ul > li {
-  border: 1px solid var(--dark-color);
-  background-color: var(--background-color)
+  position: relative;
+  background: var(--body-text-color) 0% 0% no-repeat padding-box;
+  box-shadow: 0 3px 6px #00000029;
+  border-radius: 16px;
+  opacity: 1;
+  color: var(--light-gray);
+  margin-right: 32px;
+  min-width: 245px;
+}
+
+.cards > ul > li:last-child {
+  margin-right: 0;
+}
+
+.cards.training .cards-card-body > ul {
+  /* TODO: Uncomment logic for phase 2 later, hiding duration for phase 1 */
+  display: none;
```

Comment:
@mrugesh-adobe We should hide the progress bar for now. Duration is still needed to be rendered

Reply:
Thanks @mkbansal1 .
Done.